Automattic/wp-calypso
# Components: Use BEM class names to avoid conflict

Fixes #76032.

## Proposed Changes

* Replaces the generic `.promo-card` class name with `.app-promo-card`.
* Prepends all generic class names used by the component with `.app-promo-card__`.

## Testing Instructions



* Ensure that #76032 is no longer reproducible on this PR's live branch.
* Ensure that the story for this component continues to work as expected (`yarn workspace @automattic/components storybook`).
* Ensure that the component continues to work as expected on Stats -> Traffic and Stats -> Insights pages.

## Pre-merge Checklist
